src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -547,6 +515,16 @@ export default {
       });
     }
   },
+  created() {
+    if (this.nftIsNew && this.isWritingNft) {
+      this.$router.push(
+        this.localeLocation({
+          name: 'nft-class-classId',
+          params: { classId: this.classId },
+        })
+      );
+    }
+  },
```

Comment:
If I use` this.$router.push` in `mounted()`, then there will be an obvious page transition every time the screen enters the NFT ID page, instead of an immediate redirect. In `created()`, the store's getters already have values, so it can redirect immediately.

Reply:
how about using fetch() with redirect()